site/_includes/footer.html

```diff
@@ -33,7 +33,7 @@
   <div class="footer-row" id="footer_bottom">
     <div class="footer-legal">
       <span class="copyright">
-        &copy; 2016 ManageIQ.
+        &copy; 2017 ManageIQ.
```

Comment:
@cybette Time.now.year is only the year, and I wouldn't worry about timezones over the few hours on the new year :)

@skateman's solution is more Jekylly, looks good.